etna/models/sarimax.py

```diff
@@ -6,10 +6,12 @@
 from typing import Sequence
```

Comment:
In `SARIMAX` and `AutoSARIMAX` add notes about the target components

Reply:
Added